frontend/src/pages/NewRunV2.tsx

```diff
@@ -500,11 +513,29 @@ function NewRunV2(props: NewRunV2Props) {
               }}
             />
 
+            {/* Checkbox: Always Use Latest Version*/}
+            <FormControlLabel
+              control={
+                <Checkbox
+                  checked={useLatestVersion}
+                  onChange={e => {
+                    const isChecked = e.target.checked;
+                    setUseLatestVersion(isChecked);
+                  }}
+                  disabled={!isRecurringRun || !existingPipeline}
```

Comment:
Rather than the checkbox be disabled, could you please hide it entirely if it's not a recurring run? This is consistent with the "Run trigger" section not showing when it's a one off run.